### ISSUE TITLE
[2018.3.3] fedora 27 tests hanging

### ISSUE THREAD

**Issue by Ch3LL** [saltstack/salt-jenkins#1078]
https://jenkinsci.saltstack.com/job/2018.3.3/view/Python3/job/salt-fedora-27-py3/6/

```
08:42:56        [CPU:0.0%|MEM:12.3%]  ... OK (10.349s)
08:42:56          test_exit_status_unknown_user (integration.shell.test_syndic.SyndicTest)
08:43:07        [CPU:100.0%|MEM:12.3%]  ... OK (10.737s)
08:43:07          test_issue_7754 (integration.shell.test_syndic.SyndicTest)
09:13:07 Build timed out (after 30 minutes). Marking the build as aborted.
```

**Comment by Ch3LL** [comment]
ping @terminalmage mind taking a look here if you dont have any other release tasks.

**Comment by terminalmage** [comment]
A fix was merged, this should no longer hang.

**Comment by Ch3LL** [comment]
ping @terminalmage the tests are now hanging on this test part of the test suite: `10:46:55          test_issue_7754 (integration.shell.test_syndic.SyndicTest)` could that be related to your change?

**Comment by terminalmage** [comment]
@Ch3LL and I worked on this today. It was not failing for me using the docker provisioner for test-kitchen, but @Ch3LL was able to get it to reproduce using the AWS provisioner. I'm pretty certain that https://github.com/saltstack/salt/pull/49241 will fix this, as once we applied this change to the AWS box provisioned by test-kitchen, it passed 9 times in a row (prior to this I was able to get it to hang one out of every two or three times the test was run).
